server/src/main/resources/org/elasticsearch/bootstrap/security.policy

```diff
@@ -55,6 +55,11 @@ grant codeBase "${codebase.plugin-classloader}" {
 //// Everything else:
 
 grant {
+   permission java.util.PropertyPermission "*", "write";
+
+
+  permission java.lang.RuntimePermission "accessDeclaredMembers";
```

Comment:
that was for Json layout. If we were to decide using it, we would have to leave a permission like this

Reply:
I'm super sure we don't want to grant this to the server in general. We've worked hard to remove it where possible.

Reply:
it will be gone - as discussed in [previous comment](https://github.com/elastic/elasticsearch/pull/36833#discussion_r242889744)